module/support/SoccerSimulator/src/VirtualGoalPost.cpp

```diff
@@ -166,12 +166,11 @@ namespace support {
                 result.goals.at(0).measurements.erase(result.goals.at(0).measurements.begin() + 1);
                 result.goals.at(0).measurements.erase(result.goals.at(0).measurements.begin());
             }
-            if (!result.goals.at(0).measurements.empty()) {
-                result.goals.at(0).quad.tl = convert(quad.getTopLeft());
-                result.goals.at(0).quad.tr = convert(quad.getTopRight());
-                result.goals.at(0).quad.br = convert(quad.getBottomRight());
-                result.goals.at(0).quad.bl = convert(quad.getBottomLeft());
-            }
+            // if (!result.goals.at(0).measurements.empty()) {
+            //     result.goals.at(0).post.top    = (convert(quad.getTopLeft()) + convert(quad.getTopRight()) * 0.5);
+            //     result.goals.at(0).post.bottom = (convert(quad.getBottomLeft()) + convert(quad.getBottomRight()) *
+            //     0.5);
+            // }
```

Comment:
This module should be removed in a future PR

Reply:
I dont disagree 
@TrentHouliston was this important?

Reply:
If you want to simulate stuff for localisation?